core/src/main/scala/io/gearpump/util/Constants.scala

```diff
@@ -151,4 +151,6 @@ object Constants {
   val GEARPUMP_UI_AUTHENTICATOR_CLASS = "gearpump.ui-security.authenticator"
 
   val PREFER_IPV4 = "java.net.preferIPv4Stack"
+
+  val APPLICATION_EXECUTOR_NUMBER = "gearpump.application.executor-num"
```

Comment:
If this means a new config, then we should also add this into geardefault.conf and gear.conf.